src/Http/RequestFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http;
+
+use Laminas\Diactoros\ServerRequestFactory;
+use Psr\Http\Message\ServerRequestInterface as PsrRequest;
+
+final readonly class RequestFactory
```

Comment:
Should we just make this PSR as well? We could swap this from Laminas pretty easily in the future.

Reply:
Do you mean remove this class and use laminas directly? Or something else?

Reply:
Do you mean `RequestFactory implements ServerRequestFactoryInterface` (or whatever it's called?)

Reply:
Yep.